src/box/box.cc

```diff
@@ -2834,6 +2835,28 @@ box_session_push(const char *data, const char *data_end)
 	return rc;
 }
 
+API_EXPORT int
+box_bloom_bsize(uint32_t space_id, uint32_t index_id, size_t *result) {
+	struct space *space;
+	struct index *index;
+
+	/** See check_index() for explanation. */
+	space = space_cache_find(space_id);
+	if (space == NULL)
+		return -1;
+	if (access_check_space(space, PRIV_R) != 0)
+		return -1;
+	index = index_find(space, index_id);
+	if (index == NULL)
+		return -1;
+	struct vy_lsm *lsm = vy_lsm(index);
```

Comment:
**memtx* spaces not handled.

Reply:
How can I check that space I get from `space_cache_find` is a **memtx** space?

Reply:
Should I return -1 in case of memtx engine found?

Reply:
Yep.

Reply:
Done.